cross-windows.nix

```diff
@@ -114,6 +130,7 @@ pkgs.pkgsBuildBuild.mkShell ({
     "--with-ghc=x86_64-w64-mingw32-ghc"
     "--with-ghc-pkg=x86_64-w64-mingw32-ghc-pkg"
     "--with-hsc2hs=x86_64-w64-mingw32-hsc2hs"
+    "--test-wrapper=x86_64-w64-mingw32-test-wrapper"
```

Comment:
Maybe this should have a comment, 
> We use the test-wrapper option here so that tests are executed through WINE, as we can't run windows test natively.

Reply:
Added